### ISSUE TITLE
Unable to resolve module url from ...

### ISSUE THREAD

**Issue by pachun** [jeffreymendez1993/react-native-webview-readability#16]
Running a brand new (expo, typescript, yarn for deps) app and adding the react-native-reader dependency works fine. Adding a `ReaderView` component to a render causes a crash on my system.

[Here's a new project (created with `expo init`) which reproduces the issue](https://github.com/pachun/reader-test).

Steps to reproduce:

```
git clone https://github.com/pachun/reader-test
cd reader-test
yarn install
yarn ios
```

What I see (and what I hope you'll see):

<img width="420" alt="Screen Shot 2021-04-18 at 2 26 27 PM" src="https://user-images.githubusercontent.com/1297027/115156530-aae79c00-a052-11eb-8e01-cc99e1160bf8.png">

<img width="950" alt="Screen Shot 2021-04-18 at 2 26 37 PM" src="https://user-images.githubusercontent.com/1297027/115156533-b1761380-a052-11eb-8642-0035a0b606fd.png">

What I'd like to see: A beautiful reader-type webview :D

Please let me know if I've been unclear, or if you can't reproduce the issue, or if I can help figure this out by doing anything else for you.

Thanks for the package!

**Comment by j-mendez** [comment]
@pachun thank you for the ticket, can you try v0.6.11. After installing make sure to restart the react-native metro. Seems like expo does not ship with the `url` lib but, standard rn does.

**Comment by j-mendez** [comment]
![Screen Shot 2021-04-18 at 4 54 09 PM](https://user-images.githubusercontent.com/8095978/115160533-b5139580-a066-11eb-8716-18aac1eafc6e.png) closing, fixed in `v0.6.11`

**Comment by pachun** [comment]
Thank you, @j-mendez ! Working great!
